api/Path2D.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Path2D",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "36"
```

Comment:
Confirmed by the date of https://bugs.chromium.org/p/chromium/issues/detail?id=159839#c18